src/backend/db/migrations/1_initial-migrations.sql

```diff
@@ -70,12 +71,44 @@ CREATE TABLE "goal" (
   goal_id UUID PRIMARY KEY DEFAULT uuid_generate_v4(), -- TODO: add index to allow reordering
   iep_id UUID REFERENCES "iep" (iep_id),
   description TEXT NOT NULL,
-  created_at TIMESTAMPTZ NOT NULL DEFAULT NOW()
+  created_at TIMESTAMPTZ NOT NULL DEFAULT NOW(),
+  category TEXT NOT NULL CHECK (category IN ('writing', 'reading', 'math', 'other'))
```

Comment:
🐧 convention has been to have "common" columns like `created_at` at the end of the table definition